solr/core/src/test/org/apache/solr/request/SimpleFacetsTest.java

```diff
@@ -1253,9 +1253,8 @@ public void testDateRangeFieldFacets() {
   private void helpTestDateFacets(final String fieldName, final FacetRangeMethod rangeFacetMethod) {
```

Comment:
I'm gonna be awkward here and propose renaming of the parameter as an alternative i.e. smaller diff and keeps style of the test intact with the `p/b/f/c` etc. variable names style.

```suggestion
  private void helpTestDateFacets(final String f, final FacetRangeMethod rangeFacetMethod) {
```

(I'll push change directly to the PR branch if that's alright.)

Reply:
https://github.com/apache/solr/pull/924/commits/edd62f508b69bbd8147e693dfc208f2f35ccf4e5 pushed to the branch.